src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLocaleLowerCase()
```

Comment:
you may create variable and save `query.toLowerCase().trim()` to avoid doing it twice
or even better create a function that accepts a string (title or description in this case) to compare it with the query to avoid duplicating the same code twice for title and for description